py/server/tests/test_table_factory.py

```diff
@@ -450,5 +456,19 @@ def test_input_table_empty_data(self):
         it.delete(t)
         self.assertEqual(it.size, 0)
 
+    def test_j_input_wrapping(self):
```

Comment:
Maybe a regression test for non-BaseArrayBackedInputTables too? Similar to sub test "custom input table creation".